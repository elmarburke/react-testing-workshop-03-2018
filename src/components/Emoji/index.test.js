import React from 'react';
import { Emoji } from './index';
import { shallow } from 'enzyme';

// components/Emoji/index.test.js

describe('<Emoji />', () => {
  test('render', () => {
    const tree = shallow(<Emoji shortName="unicorn" />);
    expect(tree).toMatchSnapshot();
  });
});
