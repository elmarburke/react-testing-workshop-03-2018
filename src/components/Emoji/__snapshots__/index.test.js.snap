// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Emoji /> render 1`] = `
<EmojiWrapper>
  ðŸ¦„
</EmojiWrapper>
`;
